storage/redis/client.go

```diff
@@ -209,3 +208,21 @@ func (client *Client) allPrefixedItems(prefix, first, last storage.Key) (storage
 
 	return all, nil
 }
+
+//Enqueue add a FIFO element, for DistQueue
+func (client *Client) Enqueue(value storage.Value) error {
```

Comment:
I think making a separate implementation might be better, otherwise queue and the key-value store might end up conflicting.

Reply:
moved to new file